### PR TITLE
Nit 934 alfresco content services repository deployment

### PR DESCRIPTION
**Pull request by shazChaudhry**

### CODE REVIEW

File: alfresco-content-services/values.yaml [hunk 1]
```diff
@@ -950,7 +968,7 @@ global:
     auth: secret
     # -- Shared secret to authenticate repo/solr traffic.
     # Strong enough secret can be generated with `openssl rand 20 -base64`
-    sharedsecret: null
+    sharedsecret: sharedsecret
```
Comment (pete-j-g) [inline alfresco-content-services/values.yaml:971]: if im not mistaken "sharedsecret" is simply the secret vaule in plain text. ultimately we should not be committing a secret to the repo. would it be possible to generate a kubernetes secret through a template?

alternatively, as we are still in poc stages, we could leave this as is and add a ticket to our backlog to improve this before we start dev testing/migration work

File: alfresco-content-services/values.yaml [hunk 2]
```diff
@@ -100,7 +101,7 @@ repository:
     baseSize: 20Gi
     # -- Specify a storageClass for dynamic provisioning
     accessModes:
-      - ReadWriteMany
+      - ReadWriteOnce
```
Comment (pete-j-g) [inline alfresco-content-services/values.yaml:104]: do we fully understand the consequences of changing access mode to RWO? if not we should have a backlog ticket to investigate and determine if we can use RWO here

File: alfresco-content-services/values.yaml (hunk 1 above)
Comment (pbasumatary) [reply alfresco-content-services/values.yaml:971]: Yes Pete, agree..I think we should add a ticket to our backlog to do this. For this PoC context, if we have share and cs both up and running, we'll start cleaning up in our next sprint. Is that okay?

File: alfresco-content-services/values.yaml (hunk 2 above)
Comment (pbasumatary) [reply alfresco-content-services/values.yaml:104]: yes this also needs more work so we'll add a new story in our backlog.

**Pull request merged**